Sources/SwiftInspectorVisitors/Tests/PropertyVisitorSpec.swift

```diff
@@ -336,6 +336,31 @@ final class PropertyVisitorSpec: QuickSpec {
         }
       }
 
+      context("when the property is part of a protocol definition") {
+        context("and is only gettable") {
+          let content = """
+          var foo: Foo { get }
```

Comment:
I think you want this example to be inside a protocol, if not this won't compile. In that case the property will be indeed inside a variable declaration.

```swift
protocol Some {
  var foo: Foo { get }
}
```

Reply:
I tried this at first. I then realized that we skip analyzing protocols.

![image](https://user-images.githubusercontent.com/1791049/133710702-0d917283-2d91-4990-bc84-541c8887f3b4.png)

We could invoke this visitor within a parent visitor that visits protocols.

Do you think we should do that? I expect we will end up with the same result though it could be nice to have a more realistic test.

Reply:
Oh yeah that's a good point, we should get this inside of a protocol instead and visit through the protocol instead of directly here. I don't think we'd get the same result actually, because this code as is doesn't compile so it's giving us incorrect node information (you can see that the nodes are Unknown in your screenshot on top).

Reply:
@fdiaz I made the change and we get the same result. https://github.com/fdiaz/SwiftInspector/commit/f079eac0af6e7e70e90d631b5e34af240ced918c, https://github.com/fdiaz/SwiftInspector/commit/889c6d335e581b53302032fd004fac8125e0d116

Reply:
It works now!